llama_toolchain/core/datatypes.py

```diff
@@ -151,12 +151,23 @@ def remote_provider_spec(
 
 @json_schema_type
 class DistributionSpec(BaseModel):
-    distribution_type: str
-    description: str
-
+    distribution_type: str = Field(
+        default="local",
+        description="Name of the distribution type. This can used to identify the distribution",
+    )
+    description: str = Field(
+        default="Use code from `llama_toolchain` itself to serve all llama stack APIs",
+        description="Description of the distribution",
+    )
     docker_image: Optional[str] = None
-    providers: Dict[Api, str] = Field(
-        default_factory=dict,
+    providers: Dict[str, str] = Field(
+        default={
```

Comment:
do we want a default here? In fact, we should make this required without a default